gradle.properties

```diff
@@ -9,11 +9,11 @@ releasePluginVersion=2.6.0
 puppycrawlCheckstyleVersion=8.18
 ballerinaGradlePluginVersion=1.0.0
 
-ballerinaLangVersion=2201.3.0
+ballerinaLangVersion=2201.4.0-20221116-082400-3eca35ee
```

Comment:
Shall we change this to `2201.4.0-20221208-193400-e9fb14ee`? That is the latest

Reply:
Updated in [07baf9c](https://github.com/ballerina-platform/module-ballerina-log/pull/746/commits/07baf9c1db734dfa21ce0ede5df6a8f31d9bdcb9)